### PR TITLE
added emailMessage maintain functionality

### PR DESCRIPTION
**Pull request by sollydg**

### CODE REVIEW

File: src/views/AdminEmailManage.vue [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        
+            <tr v-for="(a, index) in emails" style="background-color: #D5DFE7;">
+                <td class="pa-4">{{ a.name }}</td>
+                <td class="pa-4">{{ a.description}}</td>
+                <td class="pa-4">{{ a.text}}</td>
+                <td>
+               
+
+                    <v-btn class="mr-4" color="primary" style="float:right" @click="deleteEmailMessage(a.emailMessageId)">delete</v-btn>
+                    <v-btn class="mr-4" color="button_blue" style="float:right" @click="editEmail(a.emailMessageId)">edit</v-btn>
```
Comment (JonathanBooker912) [inline src/views/AdminEmailManage.vue:91]: Place these in a "v-row" so they display side-by-side

File: src/views/AdminEmailAdd.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                <p class="pl-5" style="font-weight: bold;">Name</p>
+                <v-text-field class="pl-5 pr-5" label="enter name for message" v-model="emailName"></v-text-field>
+            </div>
+            <div>
+                <p class="pl-5" style="font-weight: bold;">Description</p>
+                <v-text-field class="pl-5 pr-5" label="enter description" v-model="emailDesc"></v-text-field>
+            </div>
+            <div>
+                <p class="pl-5" style="font-weight: bold;">Content</p>
+                <v-text-field class="pl-5 pr-5" label="enter the content of the email" v-model="emailText"></v-text-field>
```
Comment (JonathanBooker912) [inline src/views/AdminEmailAdd.vue:45]: - Change labels to remove "enter" verbage (ie. "enter description"
- If the labels are redundant with the <p> tags above the fields, change the labels to provide more useful information, or remove them to be consistent

File: src/views/AdminEmailEdit.vue [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                <v-text-field class="pl-5 pr-5" label="enter name for message" v-model="emailName"></v-text-field>
+            </div>
+            <div>
+                <p class="pl-5" style="font-weight: bold;">Description</p>
+                <v-text-field class="pl-5 pr-5" label="enter description" v-model="emailDesc"></v-text-field>
+            </div>
+            <div>
+                <p class="pl-5" style="font-weight: bold;">Content</p>
+                <v-text-field class="pl-5 pr-5" label="enter the content of the email" v-model="emailText"></v-text-field>
+            </div>
```
Comment (JonathanBooker912) [inline src/views/AdminEmailEdit.vue:62]: Change the labels here too

**Review by JonathanBooker912 — CHANGES_REQUESTED** [review]
Everything works functionally, just some style changes. Check the failed test on agility if you need a visual for what I mean about the buttons.

**Review by DarkoPrime — APPROVED** [review]

**Pull request merged**